### PR TITLE
Remove Pinterest sharing option

### PR DESCRIPTION
**Pull request by mchv**
## What does this change?

Remove ability to share content on `Pinterest`:
- `Pinterest` is the only social platform which load an external `script` giving ability to track our readers
- We don't get significant audience back from `pinterest` 

## Does this change need to be reproduced in dotcom-rendering ?

- [ ] No
- [X] Yes — Need to raise a similar PR

## What is the value of this and can you measure success?

Remove a feature not needed from editorial perspective anymore. 
Reduce `CSS` and `JS` by side effect.

### Does this affect other platforms?

- [ ] AMP 
- [ ] Apps
- [ ] Other (please specify)

### Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

- [ ] No
- [X] Yes

### Does this change break ad-free?

- [X] No
- [ ] It did, but tests caught it and I fixed it
- [ ] It did, but there was no test coverage so I added that then fixed it

### Does this change update the version of CAPI we're using?

- [X] No, all the existing database files are just fine
- [ ] Yes, and I have [re-run all the tests locally and checked in all the updated data/database/xyz files](https://github.com/guardian/frontend/blob/main/docs/03-dev-howtos/15-updating-test-database.md)

### Tested

- [X] Test on `PROD` and rollback if needed
- [ ] Locally
- [ ] On CODE (optional)

### CODE REVIEW

**Comment by arelra** [comment]
Hi, I think there's another 'pinterest' clause here: [common/modules/gallery/lightbox.js#L147](https://github.com/guardian/frontend/blob/mc-remove-pinterest-in-lightbox/static/src/javascripts/projects/common/modules/gallery/lightbox.js#L147)
It could be my eyesight but I don't see it removed in this PR?

File: static/src/javascripts.flow.archive/bootstraps/enhanced/common.js [hunk 1]
```diff
@@ -54,7 +54,6 @@ import { init as initRelativeDates } from 'common/modules/ui/relativedates';
 import { smartAppBanner } from 'common/modules/ui/smartAppBanner';
```
Comment (ioanna0) [inline static/src/javascripts.flow.archive/bootstraps/enhanced/common.js:54]: Anything under `javascript.flow.archive` directory can remain untouched as it just serves as a snapshot for flow types

**Comment by mchv** [comment]
@arelra thanks fixed

File: static/src/javascripts.flow.archive/bootstraps/enhanced/common.js (hunk 1 above)
Comment (mchv) [reply static/src/javascripts.flow.archive/bootstraps/enhanced/common.js:54]: Yes, but what would be the benefit for not removing code that is deleted in this case? 
I find it easier to remove all references to `pinterest`.

File: static/src/javascripts/projects/common/modules/social/pinterest.js [hunk 2]
```diff
@@ -1,34 +0,0 @@
...
-        document.querySelectorAll('img:not(.gu-image):not(.responsive-img)')
-    );
-
-    fastdom.mutate(() => {
-        images.forEach(img => {
-            img.setAttribute('data-pin-nopin', 'true');
-        });
-    });
-
-    loadScript(`${scriptUrl}?r=${cachePurge}`);
```
Comment (mxdvl) [inline static/src/javascripts/projects/common/modules/social/pinterest.js:19]: Good riddance!

**Review by mxdvl — APPROVED** [review]
Pinterest? more like Boardom.

**Review by rcrphillips — APPROVED** [review]
Happy for this to happen

**Pull request merged**